Core/Utilities/Glue2.py

```diff
@@ -218,6 +219,7 @@ def __getGlue2ExecutionEnvironmentInfo(host, executionEnvironment):
               'GlueHostOperatingSystemName': osFamily,
               'GlueHostOperatingSystemRelease': osVersion,
               'GlueHostArchitecturePlatformType': architecture,
+              'GlueHostBenchmarkSI00': '2500',  # needed for the queue to be used by the sitedirector
```

Comment:
Shouldn't it be:
```
 'GlueHostBenchmarkSI00': glueHostBenchmarkSI00
```
and (few lines above):
```
glueHostBenchmarkSI00 = 2500 if not something_something
```
?

Reply:
Honestly, I don't see how the Benchmarks are connected to the computing shares, execution environments or whatever, so I have no sensible way of implementing `something_something`. And as there are no benchmarks for the regular CERN CEs...